src/rodeo.rs

```diff
@@ -109,9 +109,24 @@ pub const IndexFileMagicNumber: u32 = 0x54154170; // Shishitō
 pub const ClusterFileMagicNumber: u32 = 0xba4a4a; // Banana
 
 impl GoatRodeoCluster {
-  // Reset the index. Should be done with extreme care
-  pub fn reset_index(&self) -> () {
-    self.index.store(Arc::new(None));
+  /// get the cluster file hash
+  pub fn get_cluster_file_hash(&self) -> u64 {
+    self.cluster_file_hash
+  }
+
```

Comment:
Does the indirection get us anything if we're not relying on the ABI?

Reply:
Part of it is my Java roots... getters everywhere!

Part of it is figuring out how to abstract the data structure. Big Tent is a library and at some point we'll have to
stabilize the API... having pub stuff on structs makes API evolution harder...

Reply:
Yeah, a little harder. My experience is that most evolutions of note reshape it in more fundamental ways anyway and it comes out in the wash. (And breaking isn't the worst thing, just have to plan and communicate it a bit)